adagucserverEC/CImgRenderPolylines.cpp

```diff
@@ -143,29 +193,56 @@ void CImgRenderPolylines::render(CImageWarper *imageWarper, CDataSource *dataSou
               cnt++;
             } else {
               CDBDebug("status: %d %d [%f,%f]", status, j, tprojectedX, tprojectedY);
-              //                         dlat=CCONVERTUGRIDMESH_NODATA;
-              //                         dlon=CCONVERTUGRIDMESH_NODATA;
             }
-            //                       projectedX[j]=dlon;
-            //                       projectedY[j]=height-dlat;
           }
 
-          //                    CDBDebug("Draw polygon: %d points (%d)", cnt, numPoints);
+          // CDBDebug("Draw polygon: %d points (%d)", cnt, numPoints);
           drawImage->poly(projectedX, projectedY, cnt, drawPointLineWidth, drawPointLineColor2, drawPointLineColor2, true, false);
-          //                    break;
+          // Determine centroid for first polygon.
+          if (firstPolygon) {
```

Comment:
Would it be possible to split this extra functionalities over different files?

Not really a big issue to put it in like this, just a question.

Reply:
A bit too much work now, I feel.